### DISCUSSION TITLE
Language detection fail for polish using nova model

### DISCUSSION THREAD

**Discussion by f0enix** [deepgram/community#595] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

i am using Deepgram nodejs sdk v3.1.9 to transcribe a pre-recorded file in polish language using `nova-2-general`model. despite specifying automatic language detection, the detected language is always english with a confidence of around 0.5. According to the docs, the nova-2 model supports polish.
My request id is `319c9299-733c-4604-8fbb-ca45afd5bf34`

Other details:
-Even using the `enhanced` model gives same result
-Even if i specify the language as `pl` in the sdk, the return detected language is still `en` with blank transcription
-Using `whisper-tiny` returns the correct polish language

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

319c9299-733c-4604-8fbb-ca45afd5bf34

### If possible, please attach your code or paste it into the text box.

```javascript
 const deepgram = createClient(deepgramApiKey);
  const response = await deepgram.listen.prerecorded.transcribeUrl({ url: url }, {
    smart_format: true,
    model: 'nova-2-general',
    utterances: true,
    detect_language: true,
    diarize: false,
    detect_entities: false,
    punctuate: true
  })
```

### If possible, please attach an example audio file to reproduce the issue.
https://easyupload.io/4jplvu

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment, marked as answer]
Hey @f0enix, `nova-2-general` supports Polish but our language detection does not recognize Polish (yet!). You can see a full list of the detectable languages [here](https://developers.deepgram.com/docs/language-detection). I've passed your feedback along to our team.
